local-setup/py/worker.py

```diff
@@ -163,12 +163,12 @@ def run_in_background(self, log_file: TextIO, flags: [str] = None, subcommand_fl
         """
 
         # Todo: make this configurable
-        env = dict(os.environ, RUST_LOG='info,ws=warn,sp_io=warn,'
+        env = dict(os.environ, RUST_LOG='warn,ws=warn,sp_io=warn,'
                                         'substrate_api_client=warn,'
                                         'jsonrpsee_ws_client=warn,'
                                         'jsonrpsee_ws_server=warn,'
-                                        'enclave_runtime=info,'
-                                        'integritee_service=info,'
+                                        'enclave_runtime=warn,'
+                                        'integritee_service=warn,'
```

Comment:
It does seem to solve it, yes. The question is, are we fine with this few information in the logs?

Reply:
I guess we have to be. Better few than no logs. But maybe we should distinguish between M6 (running long) and M8 & sidechain (short).

Reply:
It's certainly better to have less detailed logs compared to none 👍 I'm not sure how simple it is to make this configurable. And now that M8 runs longer (because I've more than doubled the sleep time), it might not be certain the logs are small enough. 
Maybe at some point we can confirm with GitHub actions that there is a size limit for the logs and check if it's configurable somehow.

Reply:
That's weird, yes - and yet that's what seems to be happening. Maybe we can investigate this in more detail at some point